aws/cloud-formation/task-definition/prod-create-taskdefiniton.yaml

```diff
@@ -0,0 +1,180 @@
+            - Name: http.nonProxyHosts
+              Value: '172.17.*|localhost|127.*'
+            - Name: feedbackAssetsEndPoint
+              Value: 'https://digiroadtest.testivaylapilvi.fi/'
+            - Name: bonecp.jdbcUrl
+              Value: 'jdbc:postgresql://ddw6gldo8fiqt4.c8sq5c8rj3gu.eu-west-1.rds.amazonaws.com:5432/digiroad2'
+            - Name: bonecp.username
+              Value: 'digiroad2dbuser'
+            - Name: tierekisteri.username
+              Value: 'trrestoth'
+            - Name: tierekisteri.old.username
+              Value: 'trrest'
+            - Name: oag.username
+              Value: 'svc_clouddr'
+            - Name: emailTo
+              Value: 'info@digiroad.fi'
+            - Name: emailHost
+              Value: 'email-smtp.eu-west-1.amazonaws.com'
+            - Name: emailPort
+              Value: '587'
+            - Name: emailFrom
+              Value: 'noreply.digiroad@vaylapilvi.fi'
+            - Name: env
+              Value: 'prod'
+            - Name: caching
+              Value: 'true'
+            - Name: cacheHostname
+              Value: 'roadlinkcache.4xakoi.cfg.euw1.cache.amazonaws.com'
+            - Name: cacheHostPort
+              Value: '11211'
+            - Name: cacheTTL
+              Value: '72000'
+            - Name: rasterServiceUrl
+              Value: 'https://testioag.vayla.fi/rasteripalvelu-mml'
+          Secrets:
+            - Name: bonecp.password
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/bonecp.password'
+            - Name: tierekisteri.password
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/tierekisteri.password'
+            - Name: tierekisteri.old.password
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/tierekisteri.old.password'
+            - Name: viite.apikey
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/apikey/viite'
+            - Name: vkm.apikey
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/apikey/viitekehysmuunnin'
+            - Name: ses.username
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/ses_username'
+            - Name: ses.password
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/ses_password'
+            - Name: oag.password
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/authentication.oag.basic.password'
+            - Name: googlemapapi.client_id
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/googlemapapi.client_id'
+            - Name: googlemapapi.crypto_key
+              ValueFrom: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/prod/googlemapapi.crypto_key'
+          memory: 2048
```

Comment:
voisko vetää nää CPU ja memory hommas vetää ylemmäs, kun näitä varmaan tullaan katsomaan joten olisi ehkä mukavaa että ei tarvitse rollata koko tiedostoa.